.jenkins

```diff
@@ -44,10 +44,10 @@ pipeline {
                                 -DCMAKE_BUILD_TYPE=Release \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                 -DCMAKE_CXX_COMPILER=clang++ \
-                                -DCMAKE_CXX_FLAGS="-Werror -Wno-unknown-cuda-version -Wno-gnu-zero-variadic-macro-arguments" \
+                                -DCMAKE_CXX_FLAGS="-Werror -Wno-gnu-zero-variadic-macro-arguments -Wno-linker-warnings" \
```

Comment:
Why is it ok to drop `-Wno-unknown-cuda-version`

Reply:
Apparently, newer compiler versions don't complain about the CUDA version we are using anymore. :shrug: I tried to minimize the warnings we are ignoring here.

Reply:
For the linker warning, see https://github.com/kokkos/kokkos/pull/4244#discussion_r713038908:
```
error: linking module '/opt/sycl/lib/clang/14.0.0/../../clc/remangled-l64-signed_char.libspirv-nvptx64--nvidiacl.bc': Linking two modules of different target triples: '/opt/sycl/lib/clang/14.0.0/../../clc/remangled-l64-signed_char.libspirv-nvptx64--nvidiacl.bc' is 'nvptx64-unknown-nvidiacl' whereas '/tmp/kokkos/core/src/impl/Kokkos_CPUDiscovery.cpp' is 'nvptx64-nvidia-cuda'
```